--- ckfinder-2.4/config.php~	2013-12-16 13:09:51.017571971 +0200
+++ ckfinder-2.4/config.php	2013-12-16 13:10:26.263671492 +0200
@@ -333,3 +333,9 @@
 $config['plugin_imageresize']['smallThumb'] = '90x90';
 $config['plugin_imageresize']['mediumThumb'] = '120x120';
 $config['plugin_imageresize']['largeThumb'] = '180x180';
+
+// load additional context sensitive config
+// use webserver setenv directives to set this variable.
+if (isset($_SERVER['CKFINDER_CONFIG'])) {
+	require $_SERVER['CKFINDER_CONFIG'];
+}
